modules/nf-core/checkv/downloaddatabase/meta.yml

```
@@ -0,0 +1,47 @@
+name: "checkv_downloaddatabase"
+
+description: Construct the database necessary for checkv's quality assessment
+keywords:
+  - checkv
+  - checkm
+  - mag
+  - metagenome
+  - quality
+  - isolates
+  - virus
+  - completeness
+  - contamination
+
+tools:
+  - "checkv":
+      description: Assess the quality of metagenome-assembled viral genomes.
+      homepage: https://bitbucket.org/berkeleylab/checkv/src/master/
+      documentation: https://bitbucket.org/berkeleylab/checkv/src/master/
+      tool_dev_url: https://bitbucket.org/berkeleylab/checkv/src/master/
+      doi: "http://dx.doi.org/10.1038/s41587-020-00774-7"
+      licence: "['Modified BSD']"
+
+
+input:
+  - fasta:
+      type: file
+      pattern: "*.{fasta, fna, fa, gz, bz2, xz}"
```

```suggestion
      pattern: "*.{fasta,fna,fa,.fasta.gz,.fna.gz,.fa.gz,bz2,xz}"
```

etc., can you confirm it accepts `bz2` `xz` etc?

Also normally pattern goes after description

---

I haven't tested for it, but I assume this part works https://bitbucket.org/berkeleylab/checkv/src/master/checkv/utility.py#lines-191